examples/app_multi_node/README.md

```diff
@@ -6,32 +6,42 @@ Lightning supports makes multi-node training simple by providing a simple interf
 
 You can run the multi-node raw PyTorch by running the following commands.
 
+Here is an example where you setup spawn your processes yourself.
+
 ```bash
 lightning run app app_torch_work.py
 ```
 
+or you can use the built-in component for it.
+
+```bash
+lightning run app app_component_torch.py
+```
+
 ## Multi Node with raw PyTorch + Lite
 
 You can run the multi-node raw PyTorch and Lite by running the following commands.
 
+This removes all the boilerplate around distributed strategy by you remain in control of your loops.
+
 ```bash
-lightning run app app_lite_work.py
+lightning run app app_component_lite.py
```

Comment:
I find the naming of the files here `app_component_x.py` a bit heavy for no strong reason. Why not call these just `train_lite.py`, `train_pl.py` and `train_pytorch.py`?

Reply:
Done !